core/src/com/po/fuck/collections/EntityCollection.java

```diff
@@ -0,0 +1,27 @@
+package com.po.fuck.collections;
+
+import com.po.fuck.Entity;
+
+import java.util.ArrayList;
+
+public class EntityCollection extends SimpleCollection<Entity> {
+    public static int totalTeams = 2; ///creeps and player already added
```

Comment:
Why is it here ? What does this variable mean to our code ? And why is static, but not final ?